compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/util/BreakableMode.java

```diff
@@ -0,0 +1,34 @@
+/*
+ *  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package org.wso2.ballerinalang.compiler.util;
+
+/**
+ * Mode of transferring control flow across block statements {@code BLangFail}.
+ * Supported Types:
+ * <ul>
+ * <li>NOT_BREAKABLE - Do not transfer control flow</li>
+ * <li>BREAK_WITHIN_BLOCK - Transfer control flow to the end of current block statement</li>
+ * <li>BREAK_TO_OUTER_BLOCK - Transfer control flow to the immediate outer block statement</li>
+ * </ul>
+ * @since Swan Lake
+ */
+public enum BreakableMode {
```

Comment:
1. Shouldn't this be a inner class in block node as this always associate with a block node.
2 This only applied to transfer of constrol with ragerds to `on fail` right, I think the name should reflect that.

Reply:
Refactored in 1963173478da471955eb33f0541d7cc689d94e13